### ISSUE TITLE
E2E tests addition of check hooks for various resource types

### ISSUE THREAD

**Issue by asn1809** [RamenDR/ramen#1698]
Test cases for Failover and Relocate needs to be added using recipe with check hooks having resource type as:

- [ ] Pod
- [ ] Deployment
- [ ] Statefulset

**Comment by nirs** [comment]
So you want add new tests using recipe?

We don't have yet statefulset or pods in tests, but we have a deployment so we can start with it. Or maybe you want to have one recipe managing multiple resources?

**Comment by asn1809** [comment]
Yes, new tests needs to be added for Pods and Statefulsets. Either of the approach should be fine to either add individual TCs or a single recipe covering all types of resources.

**Comment by nirs** [comment]
@asn1809 Let's add sub-issue
- Replace ocm-ramen-samples as git repo for tests - we don't want to mix user samples (used by a human using kubectl commands) and test data (used by the test frameworks using k8s client)
- StatefullSet
- DaemonSet
- StatefulSet tests
- DaemonSet tests

Sall actionable issue are easier to track and use. This issue can be a tracker.
